util/bitstream_test.go

```diff
@@ -16,6 +16,28 @@ type testDefinition struct {
 	value  uint64 // The value we expect the function to return (64 bit to allow for future functions that extract larger ints)
 }
 
+var test1Bit = []testDefinition{
+	{testdata, 2, 0},  // testdata 0 in first byte
+	{testdata, 5, 1},  // testdata 1 in first byte
+	{testdata, 23, 1}, // testdata 1 in last bit of third byte
+	{testdata, 47, 1}, // testdata 1 in last bit of last byte
+}
+
+func TestReadByte1(t *testing.T) {
+	bs := BitStream{b: testdata, p: 80}
+
+	b, err := bs.ReadByte1()
+	assertStringsEqual(t, "Expected 1 bit at bit 80, but the byte array was only 6 bytes long", err.Error())
```

Comment:
Nitpick: Do we need the assert helper methods? Can we use `assert.Error(t, err, "string")` instead?

Reply:
I am already in the process of removing all the assert helper methods. :)